### PR TITLE
Auditissue

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
pls review

### CODE REVIEW

File: src/components/WebsiteSignIn/WebsiteSignIn.scss [hunk 1]
```diff
@@ -13,16 +13,24 @@
...
           input {
             font-family: Verdana, Geneva, Tahoma, sans-serif;
             &::placeholder {
                 font-size: 14px !important;
               }
         }
+        .loginfrm {
+            margin-top: 0px;
+        }
+        .bg {
```
Comment (ShankyRana) [inline src/components/WebsiteSignIn/WebsiteSignIn.scss]: remove

File: src/components/WebsiteSignUp/WebsiteSignUp.scss [hunk 2]
```diff
@@ -28,6 +28,9 @@
         font-size: 14px !important;
       }
     }
+    .bg {
```
Comment (ShankyRana) [inline src/components/WebsiteSignUp/WebsiteSignUp.scss]: remove

File: src/components/DashboardChannel/DashboardChannel.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  componentWillMount() {
+    window.scrollTo(0,0);
+  }
+  
+
+  channelsList = () => {
+    return (
+      <div>
+        <Row className="justify-content-around mb-5">
+          <Col md={3}  className="bx-shadow">
```
Comment (ShankyRana) [inline src/components/DashboardChannel/DashboardChannel.js]: can you optimize the code, most of the code is duplicate

File: src/components/DashboardChannel/DashboardChannel.scss [hunk 4]
```diff
@@ -0,0 +1,84 @@
+// .dashboard-container {
```
Comment (ShankyRana) [inline src/components/DashboardChannel/DashboardChannel.scss]: remove

File: src/components/DashboardChannel/DashboardChannel.js [hunk 5]
```diff
@@ -137,7 +137,7 @@ class DashboardChannel extends Component {
             <div className="content">
              
               {this.channelsList()}
-              <button className="btn btn-primary waves-effect mb-5 float-right" onClick={() => {browserHistory.push('/popupreview');}} ><i className="fi-plus"></i>&nbsp;&nbsp;Proceed</button>
+              <button className="btn btn-primary waves-effect mb-5  mt-3 float-right proceedbtn" onClick={() => {browserHistory.push('/popupreview');}} ><i className="fi-plus"></i>&nbsp;&nbsp;Proceed</button>
```
Comment (ShankyRana) [inline src/components/DashboardChannel/DashboardChannel.js]: remove {} just use  browserHistory.push('/popupreview')

File: src/components/PopupReview/PopupReview.js [hunk 6]
```diff
@@ -5,8 +5,20 @@ import { Row, Col } from 'react-bootstrap';
 
 
 class PopupReview extends Component {
-  
+  constructor() {
+    super();
+    this.state= {
+      activeClass: 1,
```
Comment (ShankyRana) [inline src/components/PopupReview/PopupReview.js]: remove trailing comma

File: src/sagas/campaign.js [hunk 7]
```diff
@@ -3,6 +3,8 @@ import * as api from 'services/api';
 import * as actions from 'ducks/campaign';
 import { load, loaded } from 'ducks/loading';
 import { toast } from 'react-toastify';
+import './toast.scss';
```
Comment (ShankyRana) [inline src/sagas/campaign.js]: why is this imported in every file?

Comment (GAUTAMRAJU15) [reply src/sagas/campaign.js]: removed from others

File: src/components/PopupReview/PopupReview.js (hunk 6 above)
Comment (GAUTAMRAJU15) [reply src/components/PopupReview/PopupReview.js]: done

File: src/sagas/campaign.js (hunk 7 above)
Comment (ShankyRana) [reply src/sagas/campaign.js]: add this to index or main file.

File: src/components/DashboardChannel/DashboardChannel.js (hunk 5 above)
Comment (GAUTAMRAJU15) [reply src/components/DashboardChannel/DashboardChannel.js]: done

File: src/sagas/campaign.js (hunk 7 above)
Comment (GAUTAMRAJU15) [reply src/sagas/campaign.js]: ok sir i put that in dashboard container.js

File: src/components/WebsiteSignIn/WebsiteSignIn.scss (hunk 1 above)
Comment (GAUTAMRAJU15) [reply src/components/WebsiteSignIn/WebsiteSignIn.scss]: removed

File: src/components/DashboardChannel/DashboardChannel.js (hunk 3 above)
Comment (GAUTAMRAJU15) [reply src/components/DashboardChannel/DashboardChannel.js]: ok sir

Comment (GAUTAMRAJU15) [reply src/components/DashboardChannel/DashboardChannel.js]: done

**Pull request closed without merging**